CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ Formatted as described on [https://keepachangelog.com](https://keepachangelog.co
 
 ## Unreleased
 
+- code formatter with Ruff ([#99](https://github.com/TeachBooks/TeachBooks/pull/99/files)).
```

Comment:
IMO listing these kind of behind-the-scenes changes in the changelog is unnecessary as it doesn't impact the functionality or the end-users at all. No need to confuse people ;)